### ISSUE TITLE
[Feature] add sleeper 

### ISSUE THREAD

**Issue by Bod9001** [unitystation/unitystation#10431]
### There is no other issue with the same feature request.

- [X] I have searched the existing issues

### Why is this feature needed/good for the game?

well is TG 2017, 
basically sleeper allows quick administering of medicine to lots of crew, but doesn't always fix High amounts of damage e.g when they are in critical

### User Story

```markdown
quick healing to the rescue?
```

**Comment by MaxIsJoe** [comment]
We should finish #6272 first before adding sleepers or statis beds because they will throw balance a lot as we've discussed before

**Comment by Bod9001** [comment]
yeah is fair enough, I just added it as an issue so we remember to do it
